### ISSUE TITLE
Mentions tooltip stuck when typed too quickly

### ISSUE THREAD

**Issue by mlewand** [ckeditor/github-writer#35]
It seems that often mentions will stuck and not notice the fact that I already done typing my issue number - it cause the editing layer to prevent space insertion.

I faced this issue more and more recently and it impacts editing experience when you have just a quick comment. It was tricky for me at first to see exactly what was going on, but here it is.

In both cases what I did was:

1. Make sure that only GH-RTE extension is active (so that other extensions do not interfere).
1. Make sure you have `5880` (plain) text copied into your clipboard.
1. Open https://github.com/ckeditor/ckeditor5/issues/6068
1. Focus the editor at the bottom.
1. Type "this issue is blocked by #<PASTE> <some random text text follows>".

For some reasons it can break with two slightly different outcomes:

## Case 1

Sticking mention tooltip, allows for typing but does not allow for spaces etc.

screencast: https://drive.google.com/file/d/1TG84-XwqIKBiqQJZ5ObVwVtZW-tZDJhT/view?usp=sharing

## Case 2

Mention tooltip sticks to the very top of the page, thus at first I was unaware what is going on and I thought that I simply can't type space at all.

screencast: https://drive.google.com/file/d/1GhStGqwS8ADZ6lsYqIGVDzSDmGfc4xe2/view?usp=sharing
